.github/workflows/release-30_publish_release_draft.yml

```diff
@@ -18,7 +18,7 @@ on:
 
 jobs:
   check-synchronization:
-    uses: paritytech-release/sync-workflows/.github/workflows/check-syncronization.yml@main
+    uses: paritytech-release/sync-workflows/.github/workflows/check-synchronization.yml@main
```

Comment:
This will need to update the pointed file:
https://github.com/paritytech-release/sync-workflows/blob/main/.github/workflows/check-syncronization.yml

Reply:
Reverting this single change is ok to me.

Otherwise we should try to ping some CI people to get their opinion I guess.